src/include/daos/object.h

```diff
@@ -281,6 +281,21 @@ daos_oclass_st_set_tgt(daos_obj_id_t oid, int tgt)
 	return oid;
 }
 
+/* check if an oid is EC obj class, and return its daos_oclass_attr */
+static inline bool
+daos_oclass_is_ec(daos_obj_id_t oid, struct daos_oclass_attr **attr)
+{
+	struct daos_oclass_attr	*oca;
+
+	oca = daos_oclass_attr_find(oid);
```

Comment:
this is a linear search, I think we should make it more efficient, e.g. use binary search. 
But this can be done in a different patch